toy-submission/main.py

```diff
@@ -39,29 +36,27 @@
 # Configure the logging module
 logging.basicConfig(level=logging.INFO)
 
-quantize = "llm.int8"
+quantize = "bnb.nf4-dq"  # 4-bit NormalFloat with Double-Quantization (see QLoRA paper)
```

Comment:
The newest and greatest quantization technique which lit-gpt supports but lit-llama doesn't.